ddtrace/propagation/_database_monitoring.py

```diff
@@ -25,23 +26,27 @@
 log = get_logger(__name__)
 
 
-def _default_sql_injector(dbm_comment, sql_statement):
-    # type: (str, str) -> str
+def default_sql_injector(dbm_comment, sql_statement):
+    # type: (str, Union[str, bytes]) -> Union[str, bytes]
     try:
+        if isinstance(sql_statement, bytes):
+            # raises a unicode error when sql statements with non utf-8 encodings
+            # are concatenated. This operation will log a warning.
+            sql_statement = sql_statement.decode("utf-8", errors="strict")
+            return  bytes(dbm_comment + sql_statement, "utf-8")
         return dbm_comment + sql_statement
-    except TypeError:
+    except (TypeError, UnicodeDecodeError, UnicodeEncodeError):
```

Comment:
```suggestion
    except (TypeError, ValueError):
```

Both unicode errors derive from `ValueError`, if we wanted to be more permissive, but this is not a requirement, being explicit and narrow for expected types is 👍🏻